xsct.c

```diff
@@ -184,8 +185,8 @@ static void sct_for_screen(Display *dpy, int screen, int temp, int fdebug)
 
 int main(int argc, char **argv)
 {
-    int i, screen, screens, temp;
-    int fdebug = 0, fhelp = 0;
+    int i, screen, screens, temp, tempd;
```

Comment:
```suggestion
    int i, screen, screens, temp;
```

you can declare and initialize `tempd` futrher down